gap/fp/freeinverse.gi

```diff
@@ -45,8 +45,11 @@ function(iter)
   FG_NextIterator := function(iter)
     local word, output, i;
 
-    NextIterator(iter);
-    word := iter!.word;
+    word := ExtRepOfObj(NextIterator(iter));
+    if IsEmpty(word) then
+      word := ExtRepOfObj(NextIterator(iter));
```

Comment:
We simply skip the word corresponding to the identity of the free group; makes sense.